packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.0.13
+
+* Fix crashes on a lot of devices.
```

Comment:
This is extremely vague; please describe the issue being fixed. You haven't mentioned either the platform (even though this is platform-specific), nor the circumstances.

Reply:
@stuartmorgan Done. Looking at https://github.com/flutter/flutter/issues/72863 , seems that nobody has a clear circumstance - it is just "many people see their customers have this crash but do not know how and why".

Reply:
https://github.com/flutter/flutter/issues/72863#issuecomment-777983748 seems pretty specific.

Reply:
@stuartmorgan done